.github/workflows/rc-test.yaml

```diff
@@ -0,0 +1,73 @@
+name: OpenMM RC Test
+on:
+  workflow_dispatch:
+
+jobs:
+  rc-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
+        openmm: ["release candidate"]
+
+    env:
+      OPENMM: ${{ matrix.openmm }}
+      OE_LICENSE: ${{ github.workspace }}/oe_license.txt
```

Comment:
Given that I'm still not **that** familiar with GHA, but, shouldn't we use `secrets.OE_LICENSE` here? Or maybe the question is, why do we have to use `github.workspace` and the PATH to the license here? Just thinking if there is a risk of the license getting exposed, no more.

Reply:
I think this is is just copied from how we do it in our main yaml, right? I think the difference is in using a secret that is defined on the org level instead of the repo level. That way we don't need to copy the same secret to every repo, and also that means when we need to update or rotate it, there is a singe place we need to do it.